packages/hoppscotch-common/src/pages/settings.vue

```diff
@@ -106,6 +106,23 @@
           </section>
         </div>
       </div>
+
+      <div
+        class="md:grid md:gap-4 md:grid-cols-3"
+        v-if="platform.ui?.additionalSettingsSections?.length"
+      >
+        <div class="p-8 md:col-span-1">
+          <h3 class="heading">{{ t("settings.additional") }}</h3>
+        </div>
```

Comment:
We don't need to display a heading for additional settings. Additional settings may be used by apps like the desktop app to provide settings entries that are not present in all versions of Hoppscotch.